src/HealthChecks.InfluxDB/InfluxDBHealthCheck.cs

```diff
@@ -0,0 +1,64 @@
+using InfluxDB.Client;
+using InfluxDB.Client.Api.Domain;
+using Microsoft.Extensions.Diagnostics.HealthChecks;
+
+namespace HealthChecks.InfluxDB
+{
+    public class InfluxDBHealthCheck  : IHealthCheck, IDisposable
+    {
+        private readonly InfluxDBClient _influxdb_client;
+
+        public InfluxDBHealthCheck(string url, string username, string password)

```

Comment:
Let's use `InfluxDBHealthCheckOptions` with single ctor in `InfluxDBHealthCheck`.

Reply:
OK